src/OBD9141sim.h

```diff
@@ -75,7 +75,7 @@ class OBD9141sim{
 
         void handleRequests();
         void answerRequest(uint8_t mode, uint8_t pid);
-
+        bool use_kwp_;
```

Comment:
Please assign this to be false in the constructor, so next to 
https://github.com/iwanders/OBD9141/blob/838bd4a46490dfa8f70e25217fd4fff337e14fa3/src/OBD9141sim.cpp#L15

Reply:
Please incorporate this feedback, other than that it looks good now, thanks for incorporating the others.